src/DocBlock/Serializer.php

```diff
@@ -98,10 +98,7 @@ public function getDocComment(DocBlock $docblock) : string
         return $comment . $indent . ' */';
     }
 
-    /**
-     * @return string
-     */
-    private function removeTrailingSpaces(string $indent, string $text)
+    private function removeTrailingSpaces(string $indent, string $text) : string
```

Comment:
Unrelated to this PR, but PSR-12 says that there should be no space before that colon. Maybe that code style should be applied throughout the codebase, after this PR.

Reply:
We applied a custom codestyle based on doctrine's coding standard. Because I like it this way. So no strict PSR-12 in here right now ;-)